sample_apps/run_example.sh

```diff
@@ -64,7 +66,7 @@ SampleApps=( "simple_app"
 # ###########################################################################
 function list_apps() {
     echo "List of sample applications you can execute with this script:"
-    echo
+    echo " "
```

Comment:
I had to add `" " `, otherwise, CI was eating up blank lines. That makes the output in CI runs a bit harder to read.